pkg/virtctl/imageupload/imageupload_test.go

```diff
@@ -586,3 +587,26 @@ var _ = Describe("ImageUpload", func() {
 		)
 	})
 })
+
+type atomicBool struct {
+	lock  *sync.Mutex
+	value bool
+}
+
+func (b *atomicBool) IsTrue() bool {
+	b.lock.Lock()
+	defer b.lock.Unlock()
+	return b.value
+}
+
+func (b *atomicBool) True() {
```

Comment:
Maybe SetTrue would be a better name here ?  
Or maybe combine the True() and False() into a   func (b *atomicBool) Set(b bool)